components/wb-fieldflow/fieldflow.js

```diff
@@ -146,8 +149,8 @@ var componentName = "wb-fieldflow",
 				}
 				$( formElm.parentElement ).addClass( formComponent );
 			} else if ( config.inline && !config.renderas ) {
-				stdOut = "<div class='wb-frmvld " + formComponent + "'><form><div class='input-group'><div id='" + bodyID + "'>";
-				stdOut = stdOut + "</div><span class='input-group-btn'><input type=\"submit\" value=\"" + i18n.btn + "\" class=\"btn btn-default mrgn-bttm-md\" /></span></div> </form></div>";
+				stdOut = "<div class='wb-frmvld " + formComponent + ( showLabel ? " mrgn-bttm-md" : "" ) + "'><form><div class='input-group'><div id='" + bodyID + "'>";
```

Comment:
This short-hand if should be added to the `.input-group` element at line 149 instead since it has an unwanted visual effect where the margin bottom is not taken into account:
<img width="609" alt="fieldflow" src="https://user-images.githubusercontent.com/4353199/151227768-03ca3c27-96b8-47f4-88c3-2df27cef81d8.png">

Line 149 should become:
`<div class='input-group" + ( showLabel ? " mrgn-bttm-md" : "" ) + "'>`

Reply:
Fixed with a small different approach. Thanks!
![wet171](https://user-images.githubusercontent.com/24379236/151582745-2fca35a9-60bb-45bb-a895-d891f15033d8.png)
s